src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,17 @@
 package calculator;
 
+import calculator.delimiter.CustomDelimiterPattern;
+import calculator.delimiter.DefaultDelimiterPattern;
+import calculator.delimiter.DelimiterPattern;
+import calculator.view.InputView;
+import calculator.view.OutputView;
+import java.util.List;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        List<DelimiterPattern> patterns = List.of(new DefaultDelimiterPattern(), new CustomDelimiterPattern());
+        CalculatorService service = new CalculatorService(new InputView(), new OutputView(),
+                new InputValidator(patterns), new NumberExtractor(), new SumCalculator());
```

Comment:
```suggestion
InputView inputView = new InputView();
OutputView outputView = new OutputView();
InputValidator validator = new InputValidator(patterns);
NumberExtractor extractor = new NumberExtractor();
SumCalculator calculator = new SumCalculator();

CalculatorService service = new CalculatorService(inputView, outputView, validator, extractor, calculator);
        
```
이렇게 나눠서 쓰는 게 더 가독성에 좋을 것 같습니다!

Reply:
좋은 의견 감사합니다!